api/schema/types.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Create a separate package under heapster and place all the derived logic in there. Let the 'api' package only deal with REST endpoints.